### PR TITLE
feat: /score: support negative han/fu + honbas

### PR DESCRIPTION
**Pull request by m4tx**
image
image

I can't believe we're doing this

### CODE REVIEW

File: src/slash_commands/score.rs [hunk 1]
```diff
@@ -88,10 +106,11 @@ impl SlashCommand for ScoreCommand {
             _ => Err(format!("Invalid mode: {mode}")),
         }?;
 
-        let han = Han::new(u32::try_from(han)?);
-        let fu = Fu::new(u32::try_from(fu)?);
-        let points = Points::from_calculated(points_calculation_mode, han, fu)?;
-        let embed = create_points_embed(han, fu, &points)?;
+        let han = Han::new(i32::try_from(han)?);
+        let fu = Fu::new(i32::try_from(fu)?);
+        let honbas = Honbas::new(i32::try_from(honbas)?);
```
Comment (Iipin) [inline src/slash_commands/score.rs:111]: negative honbas 
image

**Review by Iipin — APPROVED** [review]

File: src/slash_commands/score.rs (hunk 1 above)
Comment (m4tx) [reply src/slash_commands/score.rs:111]: I also thought negative han doesn't make sense, then these Saki card players come and say "wololo we had to manually create the score table".

This is future proof (and not allowed in the default calculation mode anyway).

Comment (phoe) [reply src/slash_commands/score.rs:111]: We are making this work for the future generations of chombers to come. We are creating legacy, friend.

File: src/slash_commands/score.rs [hunk 2]
```diff
@@ -16,16 +16,22 @@ use crate::Chombot;
...
 const DEFAULT_MODE: &str = "default";
 const LOOSE_MODE: &str = "loose";
 const UNLIMITED_MODE: &str = "unlimited";
 
+const MIN_HAN: i64 = -1600;
 const MAX_HAN: i64 = 1600;
+const MIN_FU: i64 = -100000;
 const MAX_FU: i64 = 100000;
+const MIN_HONBAS: i64 = -10000;
+const MAX_HONBAS: i64 = 10000;
```
Comment (phoe) [inline src/slash_commands/score.rs:31]: :xD:

**Review by phoe — APPROVED** [review]

File: src/slash_commands/score.rs (hunk 1 above)
Comment (Iipin) [reply src/slash_commands/score.rs:111]: I am so proud to be a part of this community

**Pull request merged**